src/transforms/remap.rs

```diff
@@ -129,6 +130,7 @@ pub struct RemapConfig {
     /// to a specially-named output, `dropped`. The original event is annotated with additional
     /// fields describing why the event was dropped.
     #[serde(default = "crate::serde::default_false")]
+    #[configurable(metadata(docs::human_name = "Drop on Error"))]
```

Comment:
Should this be `Reroute Dropped Output` or similar?

Reply:
Agreed, I think this line should be up at https://github.com/vectordotdev/vector/pull/17517/files#diff-ad5088e389a2b5ae3b514738658923d21b23dda2de8b29cfb4022c73cd0f8bd2L105

Reply:
@silversupreme @spencergilbert I think I finally got this right. I used `Reroute Dropped Events`. Added `Drop Event on Error` to up where it should be. And also updated the `Drop on Abort` to `Drop Event on Abort`. Let me know if that looks good, thanks!